src/experiment/experiment.js

```diff
@@ -60,11 +57,6 @@ export function buildExperiment() {
 
```

Comment:
Remove the jsPsychForURL line. Were initializing it in the jsPsych.js. The rest can you cross reference with serve js for what we need to keep and grab it that way (vs using jsPsych methods).